spec/browser/middlewares/retry.spec.js

```diff
@@ -24,8 +25,9 @@ describe('Middleware / RetryMiddleware', () => {
     retries = 3
     headerRetryCount = 'X-Mappersmith-Retry-Count'
     headerRetryTime = 'X-Mappersmith-Retry-Time'
+    validateRetry = (response) => response.responseStatus !== 401
```

Comment:
Maybe the default should also check for the HTTP method?
I'd argue that you might not want to retry POSTs?

Reply:
The middleware only retries GET requests, if it's a GET request it proceeds and can be further validated

Reply:
https://github.com/tulios/mappersmith/blob/1eba1e571ec0cca0b6db0cca81b5c3a513711d3e/src/middleware/retry.js#L46